### ISSUE TITLE
How to restrict popup to current device?

### ISSUE THREAD

**Issue by totalitarian** [thomasloven/hass-browser_mod#341]
I assumed I could use this

```
    tap_action:
      action: call-service
      service: browser_mod.more_info
      service_data:
        entity_id: camera.side
        deviceid:
          - this
```

but the popup still happens on all device.

**Comment by jawn22** [comment]
Just read through https://github.com/thomasloven/hass-browser_mod/releases/tag/1.3.0 and it seems you need to use a fire-dom-event. 

**What I'm trying to do:**
I'm trying to use chip cards to have a home/back/rooms selector menu button. The room selector menu button will allow me to navigate to a different dashboard - as each dashboard is designed for that room. It's still a WIP but getting there slowly.

When clicking the room selector, I only want that to display on the current device.

**Current room selector popup (shows on all devices):**

square: false
columns: 1
type: grid
cards:
  - square: false
    columns: 2
    type: grid
    cards:
      - type: custom:mushroom-chips-card
        chips:
          - type: action
            icon: mdi:home
            tap_action:
              action: navigate
              navigation_path: home
          - type: back
          - type: action
            tap_action:
              action: call-service
              service: browser_mod.popup
              target: {}
              data:
                title: Rooms Navigator
                large: false
                card:
                  square: false
                  type: grid
                  columns: 2
                  cards:
                    - type: button
                      name: Bedroom
                      icon: mdi:bed
                      tap_action:
                        action: navigate
                        navigation_path: /lovelace-bedroom/home
                      hold_action:
                        action: none
                    - type: button
                      name: Tack Room
                      icon: mdi:office-building
                      tap_action:
                        action: navigate
                        navigation_path: /control-4-test/home
                      hold_action:
                        action: none
                    - type: button
                      name: Test1
                      icon: mdi:office-building
                      tap_action:
                        action: navigate
                        navigation_path: /control-4-test/home
                      hold_action:
                        action: none
                    - type: button
                      name: Test2
                      icon: mdi:office-building
                      tap_action:
                        action: navigate
                        navigation_path: /control-4-test/home
                      hold_action:
                        action: none
            icon: mdi:menu
        alignment: start



**What I tried (just with a button card as I was ruling out the chips card as the problem):**

show_name: true
show_icon: true
type: button
tap_action:
  action: fire-dom-event
  browser_mod:
    command: popup
    service_data:
      deviceID: this
      title: Rooms Navigator
      large: false
      card:
        square: false
        type: grid
        columns: 2
        cards:
          - type: button
            name: Bedroom
            icon: mdi:bed
entity: input_select.rooms
name: Rooms

**Comment by thomasloven** [comment]
Closed because of Browser Mod 2.0 release.
Please see #354.
